src/Sylius/Behat/Page/Admin/Order/ShowPage.php

```diff
@@ -422,10 +426,12 @@ protected function getDefinedElements(): array
             'ip_address' => '#ipAddress',
             'items_total' => '#items-total',
             'order_notes' => '#sylius-order-notes',
-            'order_payment_state' => '#payment-state > span',
+            'order_payment_state' => '[data-test-order-payment-state]',
             'order_shipping_state' => '#shipping-state > span',
             'order_state' => '#sylius-order-state',
-            'payments' => '#sylius-payments',
+            'payments' => '[data-test-payments]',
+            'payment_complete' => '[data-test-complete-payment="%paymentId%"]',
+            'payment_refund' => '[data-test-refund-payment="%paymentId%"]',
```

Comment:
```suggestion
            'payment_complete' => '[data-test-payment-complete="%paymentId%"]',
            'payment_refund' => '[data-test-payment-refund="%paymentId%"]',
```
Wdyt? Not a big deal, but it would be nice to adjust this for better code readability.